introduction.md

```diff
@@ -0,0 +1 @@
+Hi My name is anthofan and I am still new to this
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.